### PR TITLE
Issue #17 - Implemented tag filtering and made some minor modificatio…

### PR DESCRIPTION
**Pull request by viktorija-panovska**
…ns to the frontend elements

### CODE REVIEW

**Comment by viktorija-panovska** [comment]
What do you think about making the filtering so that it filters as the user is typing, not requiring them to press search?

**Review by barjin — APPROVED** [review]
Looking good to me - once the linter is passing, we can imo merge. Thanks for the typo fixes too. 

One small nit: Let's not commit the sqlite database changes, as that would blow up the github history quite a lot (it's binary anyway, so tracking the changes doesn't make much sense). We can of course commit if there are some changes to the database schema :)

**Comment by barjin** [comment]
Ah, regarding the incremental search - sounds cool, if you want to try

**Comment by viktorija-panovska** [comment]
Is there a way I can remove the database from this commit?

**Comment by barjin** [comment]
Something like this should work?
```bash
git reset HEAD^ -- ./packages/backend/db.sqlite
git commit --amend --no-edit

git push -f origin/tagFilter
```

`reset` brings the file back to the specified commit (`HEAD^` is one commit before), `commit --amend` appends the changes to the latest commit and `push -f` sends the commit to the server (force because we want to overwrite what's on the server).

**Comment by viktorija-panovska** [comment]
> Something like this should work?
> 
> ```shell
> git reset HEAD^ -- ./packages/backend/db.sqlite
> git commit --amend --no-edit
> 
> git push -f origin/tagFilter
> ```
> 
> `reset` brings the file back to the specified commit (`HEAD^` is one commit before), `commit --amend` appends the changes to the latest commit and `push -f` sends the commit to the server (force because we want to overwrite what's on the server).

I ran this but idk if it worked, the file still shows in changes.

File: packages/frontend/src/components/WebsiteRecordData.tsx [hunk 1]
```diff
@@ -32,6 +32,8 @@ export function WebsiteRecordData(): JSX.Element {
       });
   }, [api]);
 
+  console.log(record);
+
```
Comment (barjin) [inline packages/frontend/src/components/WebsiteRecordData.tsx:36]:
```suggestion
```

**Pull request merged**